phac-nml/irida-next
# [ENHC0010034] List samples within export dialog

## What does this PR do and why?
Added a scrollable list within the sample data export dialog to display the selected samples.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/ee097aca-ce49-4c5b-aad2-48df5c2465f1)
![image](https://github.com/phac-nml/irida-next/assets/325703/6a0b82f2-3ba3-4402-83f7-b01effa86b4e)

## How to set up and validate locally
1. Open a rails console and run the following:
 
```
project = Project.first
 
11.upto(10000) do |i|
  sample = Samples::CreateService.new(
    project.creator, project,
    { name: "#{project.namespace.parent.name}/#{project.name} Sample #{i}",
      description: "This is a description for sample #{project.namespace.parent.name}/#{project.name} Sample #{i}." 
    }
  ).execute
end
```
2. Login as admin and navigate to [http://localhost:3000/bacillus/bacillus-anthracis/outbreak-2022/-/samples](http://localhost:3000/bacillus/bacillus-anthracis/outbreak-2022/-/samples).
3. Select all the samples within the table.
4. Click the `Create Export` button.
5. Verify the selected samples are displayed.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
